libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -302,6 +306,26 @@ export type DependentEntitiesGraphResponse = DependentEntitiesResponse;
  */
 export type WorkspaceDataFiltersLayout = DeclarativeWorkspaceDataFilters;
 
+/**
+ * @internal
+ */
+export type WorkspaceDataFiltersResult = JsonApiWorkspaceDataFilterOutDocument;
+
+/**
+ * @internal
+ */
+export type WorkspaceDataFilterSettingsResult = JsonApiWorkspaceDataFilterSettingOutDocument;
+
+/**
+ * @internal
+ */
+export type WorkspaceDataFilter = JsonApiWorkspaceDataFilterInDocument;
+
+/**
+ * @internal
+ */
+export type WorkspaceDataFilterSettings = JsonApiWorkspaceDataFilterSettingInDocument;
```

Comment:
Please rename `WorkspaceDataFilterSettings` → `WorkspaceDataFilterSetting` (without any `s` after `Setting`).

`WorkspaceDataFilterSettings` can make people think that the type is some sort of array, can't it?